site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -270,7 +270,7 @@ the steps in [Viewing Warnings][27].
     <tr id="management">
       <td><code>"management"</code></td>
       <td>Grants the extension access to the <a href="/docs/extensions/reference/management">chrome.management</a> API.</td>
-      <td><strong>Manage your apps, extensions, and themes</strong></td>
```

Comment:
This really does let extensions manage apps, too, so I kind of think we should leave it in

Reply:
I put it back.